BrainPortal/app/controllers/sessions_controller.rb

```diff
@@ -51,7 +51,7 @@ def new #:nodoc:
   end
 
   def create #:nodoc:
-    if request.format.to_sym !~ /^(json|xml)$/i # JSON is used for API calls; XML not yet fully supported
+    if ! api_request? # JSON is used for API calls; XML not yet fully supported
```

Comment:
Is this comment still true, because it would api_request? will not screen agains it being XML or JSON... right?

Reply:
It's a reminder that there are controller actions where we properly return json results, but not in XML... I have tried as much as possible to provide both whenever I noticed the XML support was lacking.

Reply:
Thanks.